build.assets/Makefile

```diff
@@ -381,6 +439,52 @@ enter-root: buildbox
 		-e HOME=$(SRCDIR)/build.assets -w $(SRCDIR) $(BUILDBOX) /bin/bash
 
 #
+# Reconfigure crosstool-NG for the given ARCH via its menuconfig system.
+# e.g.
+# $ make config/buildbox-ng CROSSTOOLNG_ARCH=arm64
+#
+# After saving and exiting in the menuconfig, you should have an updated
+# defconfig file in build.assets/buildbox/ct-ng-configs
+#
+# You need to have a local copy of the configuration container image
+# created with: make buildbox-thirdparty-crosstoolng
+#
+.PHONY: config/buildbox-ng
+config/buildbox-ng:
+	@: $(or $(CROSSTOOLNG_ARCH),$(error CROSSTOOLNG_ARCH variable must be set))
+	@: $(or $(wildcard buildbox/ct-ng-configs/$(CROSSTOOLNG_ARCH).defconfig),\
```

Comment:
```suggestion
	@: $(or $(wildcard buildbox/crosstoolng-config/$(CROSSTOOLNG_ARCH).defconfig),\
```

Reply:
Yeah, I'm sorry about that. I did the renaming suggested and stuffed up a few bits and wanted to get the PR updated before I went to sleep. I usually kick off a build before pushing, but these changes were going to take over an hour to build. I'll ensure I build before I push next time.

Anyway, fixed.